fipy/meshes/gmshMesh.py

```diff
@@ -1387,6 +1389,11 @@ def _localOverlappingCellIDs(self):
 class Gmsh2D(Mesh2D):
     """Construct a 2D Mesh using Gmsh
 
+    If called in parallel, mesh will be partitioned based on
```

Comment:
If called in parallel, the mesh will be partitioned based on the value of parallelComm.Nproc

Reply:
Better, thanks